File: Selenium/Activities/Activity12.java
package activities;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity12 {

	public static void main(String[] args) {

		WebDriver driver =  new FirefoxDriver();
		
		driver.get("https://v1.training-support.net/selenium/dynamic-controls");
		
		System.out.println(driver.getTitle());
		
		WebElement textField = driver.findElement(By.xpath("//div[@id='dynamicText']"));
		
		System.out.println("text field is enabled: "+textField.isEnabled());
		
		driver.findElement(By.xpath("//button[contains(text(),'Enable Input')]")).click();
		
		System.out.println("text field is enabled: "+textField.isEnabled());
		
		driver.close();

	}

}

File: Selenium/Activities/Activity12.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Firefox() as driver:
    
    driver.get("https://v1.training-support.net/selenium/dynamic-controls")

    print(driver.title)

    textField = driver.find_element(By.XPATH, "//div[@id='dynamicText']")

    print("Text field is enabled: ", textField.is_enabled())

    driver.find_element(By.XPATH,"//button[contains(text(),'Enable Input')]").click()

    print("Text field is enabled: ", textField.is_enabled())

    driver.close()

